src/box/lua/iproto.c

```diff
@@ -268,16 +299,124 @@ push_iproto_protocol_features(struct lua_State *L)
 					  lengthof(features));
 }
 
+/**
+ * Encodes Lua table packet part to MsgPack using IPROTO key translation table:
+ * uses @a ibuf to store the result and saves the result length in @a mp_len.
+ */
+static const char *
+encode_lua_table_packet_part_to_mp(struct lua_State *L, int idx, size_t *mp_len)
```

Comment:
The name is awkward. Since this function is called from just one place, which by the way doesn't have much else to it, I suggest to inline it.

Reply:
I agree, inlined it.